### PR TITLE
feat: add textbook

### PR DESCRIPTION
**Pull request by DubichTonya**
1. Добавлена страница учебника.
2. Добавлен роутинг на страницу учебника и настройки.

### CODE REVIEW

File: src/app/redux/models/textbook.model.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  textMeaning: string;
+  textExample: string;
+  transcription: string;
+  wordTranslate: string;
+  textMeaningTranslate: string;
+  textExampleTranslate: string;
+}
+
+export interface TextbookState {
+  words: IWord[];
```
Comment (rss-maksim) [inline src/app/redux/models/textbook.model.ts:17]: Let's remove commented lines please

File: src/app/redux/reducers/textbookReducer.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  loadWordsSuccess,
+  setGroupWords,
+  setPageWords,
+  setWordSettingsAddButtons,
+  setWordSettingsTranslation,
+} from '../actions/textbooks.actions';
+
+export const initialState: TextbookState = {
+  words: [],
+  // wordsGroup: '0',
```
Comment (rss-maksim) [inline src/app/redux/reducers/textbookReducer.ts:13]: Let's remove commented lines please

File: src/app/textbook/components/textbook-menu/textbook-menu.component.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+@Component({
+  selector: 'app-textbook-menu',
+  templateUrl: './textbook-menu.component.html',
+  styleUrls: ['./textbook-menu.component.scss'],
+})
+export class TextbookMenuComponent {
+  isCheckedTranslation = false;
+  isCheckedAddButtons = false;
+  labelPosition: 'before' | 'after' = 'after';
```
Comment (rss-maksim) [inline src/app/textbook/components/textbook-menu/textbook-menu.component.ts:19]: It's better to create separate type for example ```type Position = 'before' | 'after'```

**Review by rss-maksim — APPROVED** [review]

File: src/app/textbook/components/textbook-menu/textbook-menu.component.ts (hunk 3 above)
Comment (DubichTonya) [reply src/app/textbook/components/textbook-menu/textbook-menu.component.ts:19]: Done.

**Review by IlyaKozak — APPROVED** [review]

**Pull request merged**